suite.go

```diff
@@ -383,7 +383,7 @@ func (s *Suite) matchStepText(text string) *StepDef {
 	return nil
 }
 
-func (s *Suite) runSteps(steps []*gherkin.Step) (err error) {
+func (s *Suite) RunSteps(steps []*gherkin.Step) (err error) {
```

Comment:
how about creating the public function which calls the internal one?
IMO you're missing the docs as well :)